Client/Ecosia/Migration/EcosiaImport+PerformanceTests.swift

```diff
@@ -110,3 +71,57 @@ extension EcosiaImport {
 
 }
 #endif
+
+extension EcosiaImport {
+    static func mockHistory(days: Int, visits: Int) -> [(Date, Core.Page)]  {
+        let topSiteUrls = FaviconFetcher.getTopSiteURLs()
+
+        let items: [(Date, Core.Page)] = (0..<days).map { day in
+            let now = Date()
+            let oneDay = 24 * 60 * 60
+
+            let visitsPerDay: [(Date, Page)] = (0..<visits).map { visit in
+                let url = URL(string: topSiteUrls.randomElement()!)!
+                let page = Page(url: url, title: url.host ?? url.absoluteString)
+                return (Date(timeInterval: -Double(day * oneDay + visit), since: now), page)
+            }
+            return visitsPerDay
+        }.reduce([], +)
+        return items
+    }
+
+    static func createMigrationData() {
+
+        let history = Core.History()
+        let favs = Core.Favourites()
+
+        let items = mockHistory(days: 100, visits: 50) // 100 TLDs
+        history.setItems(items)
+
+        let topSiteUrls = FaviconFetcher.getTopSiteURLs()
+
+        favs.items = (0...1000).map({ _ in
+            let url = URL(string: topSiteUrls.randomElement()!)!
+            return Page(url: url, title: url.host ?? url.absoluteString)
+        })
+
+        Core.User.shared.migrated = false
+    }
+}
+
+extension FaviconFetcher {
+    class func getTopSiteURLs() -> [String] {
+        let filePath = Bundle.main.path(forResource: "top_sites", ofType: "json")
+        let file = try! Data(contentsOf: URL(fileURLWithPath: filePath!))
+        let decoder = JSONDecoder()
+        var decoded = [BundledIcon]()
+        do {
+            decoded = try decoder.decode([BundledIcon].self, from: file)
+        } catch {
+            print(error)
```

Comment:
Is this really necessary? this is inside a test right? maybe no need to add defensive code in here?

Reply:
not needed.. was copy-paste artefact.. will remove

Reply:
@ecotopian  Sorry if my comment was not explicit enough, I meant the whole `try` `catch`, not only the `print`. Maybe we want to force try it with `try!` and if it fails it should be an alarm for us that something is not behaving as we expect?

Reply:
But is just an option

Reply:
Ah now I see your point.. fine to simplify 💪